lib/ui/painting/image_decoder_impeller.cc

```diff
@@ -421,43 +440,6 @@ ImageDecoderImpeller::UploadTextureToStorage(
   }
 
   texture->SetLabel(impeller::SPrintF("ui.Image(%p)", texture.get()).c_str());
-
```

Comment:
Since this function is only used for gifs now we don't need to worry about mip support.